source/_posts/2020-01-15-release-104.markdown

```diff
@@ -74,28 +74,165 @@ Experiencing issues introduced by this release? Please report them in our [issue
+  - SERVICE_ICLOUD_PLAY_SOUND = "play_sound"
+  - SERVICE_ICLOUD_DISPLAY_MESSAGE = "display_message"
+  - SERVICE_ICLOUD_LOST_DEVICE = "lost_device"
+  - SERVICE_ICLOUD_UPDATE = "update"
+
+  These services have been renamed:
+  - 'icloud_lost_device' --> 'lost_device'
+  - 'icloud_update' --> 'update'
+  
+  'icloud_reset' has been removed
+
+  The iCloud cookie folder moved from `[HA_dir]/icloud` to `[HA_dir]/.storage/icloud`.
+- __Pi-hole__ - The default value for host was removed, and now must be specified manually. - ([@johnluetke] - [#27569]) ([pi_hole docs]) 
+    
+    Example configuration:
+  ```yaml
+  pi_hole:
+    - host: pi.hole
+      ssl: false
+    - host: backup.pi.hole
+  ```
+
+- __HERE Travel Time__ - HERE has deprecated the usage of app_code. New users can only generate an api_key. Existing users have to follow the migration guide in order to retrieve the now needed api_key. - ([@eifinger] - [#29966]) ([here_travel_time docs])
+    
+    Example configuration:
+  ```yaml
+  sensor:
+    - platform: here_travel_time
+      api_key: "YOUR_API_KEY"
+      origin_latitude: "51.222975"
+      origin_longitude: "9.267577"
+      destination_latitude: "51.257430"
+      destination_longitude: "9.335892"
+  ```
+
+- __ZHA__ - The default divisor for active_power has changed.  Now, multiplier and divisor both default to 1, if the device does not support these attributes or we fail to get those attributes from the device. - ([@Adminiuga] - [#30130]) ([zha docs]) 
+- __Keba__ - The unique_ids of the entities have changed and are now more future proof. The default name and entity_id of the entities have also changed. Users need to clear stale entries from the entity registry and update concerned automations and scripts. - ([@dannerph] - [#30125]) ([keba docs]) 
+- __Tesla__ - Config flow has been added for Tesla. Device tracker entity names have changed due to the conversion to config_flow. They now follow the naming convention of the rest of the Tesla entities. The `id` device tracker entity attribute was also removed because it is a duplicate of `trackr_id`. - ([@alandtse] - [#28744]) ([tesla docs]) 
+- __Huawei LTE__ - The Huawei LTE notify service name is now configurable. The default name for the Huawei LTE notify service has changed to `notify.huawei_lte` to avoid conflicts with other notify services. An option to provide a name for the service has also been added. - ([@scop] - [#30208]) ([huawei_lte docs])
+
+   Example configuration:
+   ```yaml
+   huawei_lte:
+     - url: ...
+       password: ...
+       notify:
+         name: sms
+   ```
+
+- __Amcrest__ - Removed Amcrest sensors and switches that were deprecated in previous releases. Warnings have been in place since deprecated so, although this is technically a breaking change, any users of this integration, that have been using the deprecated features, should have had plenty of time to remove them from their configuration. - ([@pnbruckner] - [#30308]) ([amcrest docs])
```

Comment:
Can we remove the line "Warnings…their configuration.". It doesn't add any extra info and feels a bit passive aggressive.